### ISSUE TITLE
[BUG] Font not changing

### ISSUE THREAD

**Issue by killerdevildog11** [SheerSt/pokewilds#193]
After reading an unknown sign, the font said something in the unknown letter font. I picked up a few pokeballs on the beach and its still using the unknown font and music stilll going too from the sign.

**Comment by Squillbill** [comment]
Also persists for the battle text, though not the move names. Not sure why it reverted the first time, but disappeared after I ran into an unknow the second time, maybe relevant.

If it's a feature, there's also an issue where Regular x's appear, for example the sign about taking pokemon eggs reads Agressivexxx, and during leveling up, x's replace the level.

**Comment by Harmonia2Side** [comment]
Just had this exact issue

**Comment by Harmonia2Side** [comment]
Suggestion: Renaming issue to "Font not changing back to normal after reading unown sign"

**Comment by CaSe2474** [comment]
> After reading an unknown sign, the font said something in the unknown letter font. I picked up a few pokeballs on the beach and its still using the unknown font and music stilll going too from the sign.

the sign just had a trainer tip on it

**Comment by CaSe2474** [comment]
btw it happened to me too

**Comment by kutsen39** [comment]
This is not a bug, it's a feature! When you get an Unown sign, go into tall grass to encounter an Unown. After catching (recommended for rare candies) or defeating, everything will go back to normal.

My apologies if you did this, no one's comment mentioned getting the Unown encounter. As for the X's, there are no numbers or apostrophe in the Unown alphabet.

**Comment by CaSe2474** [comment]
> 

Although does it persist after a battle that's NOT against unown?

**Comment by kutsen39** [comment]
Yes. I've noticed that you can't fish up an Unown, so that doesn't count. I'm not sure if they only spawn in tall grass or if deep sand can trigger it. It might also revert if you leave the area and come back.

**Comment by CaSe2474** [comment]
> Yes. I've noticed that you can't fish up an Unown, so that doesn't count. I'm not sure if they only spawn in tall grass or if deep sand can trigger it. It might also revert if you leave the area and come back.

It can be encountered as one of the ruins random encounter
